lib/thunderbird_common.pm

```diff
@@ -42,61 +42,154 @@ sub tb_setup_account {
-        if (is_tumbleweed) {
-            assert_and_click 'thunderbird_SSL_pop3-selection-click-TW';
-            assert_and_click 'thunderbird_SSL_auth_click';
-            assert_and_click 'thunderbird_wizard-pop-done';
-        }
-        else {
-            # If use multimachine, select correct needles to configure thunderbird.
-            if ($hostname eq 'client') {
+    if (check_screen 'thunderbird-new-gui') {
+        wait_still_screen(2, 4);
+        type_string "SUSE Test";
+        send_key 'tab';
+        wait_screen_change { type_string "$mail_box" };
+        send_key 'tab';
+        wait_screen_change { type_string "$mail_passwd" };
+        wait_still_screen(2, 4);
+        send_key 'tab';
+        send_key 'tab';
+        send_key 'spc';    # configure manually
+        wait_still_screen(2, 4);
+        save_screenshot;
+        send_key 'tab';    # scroll page down to see configuration options
+        assert_and_click 'thunderbird_know-your-rights';
+
+        if ($proto eq 'pop') {
+            assert_and_click 'thunderbird_wizard-imap-selected';
+            assert_and_click 'thunderbird_wizard-imap-pop-open';
+            if (is_tumbleweed) {
+                assert_and_click 'thunderbird_SSL_pop3-selection-click-TW';
                 assert_and_click 'thunderbird_SSL_auth_click';
-                wait_still_screen(2);
-                send_key 'down';
-                send_key 'ret';
+                assert_and_click 'thunderbird_wizard-pop-done';
+            }
+            else {
+                # If use multimachine, select correct needles to configure thunderbird.
+                if ($hostname eq 'client') {
+                    assert_and_click 'thunderbird_SSL_auth_click';
+                    wait_still_screen(2);
+                    send_key 'down';
+                    send_key 'ret';
+                }
+                assert_screen "thunderbird_wizard-$proto-selected";
             }
-            assert_screen "thunderbird_wizard-$proto-selected";
         }
-    }
 
-    # If use multimachine, select correct needles to configure thunderbird.
-    if ($hostname eq 'client') {
-        send_key_until_needlematch 'thunderbird_SSL_done_config', 'alt-t', 4, 2;
-        wait_still_screen(2);
-        assert_and_click "thunderbird_SSL_done_config";
-        wait_still_screen(3);
-        assert_and_click 'thunderbird_SSL_done_config' unless check_screen('thunderbird_confirm_security_exception');
-        assert_and_click "thunderbird_confirm_security_exception";
-        assert_and_click "thunderbird_skip-system-integration";
-        assert_and_click "thunderbird_get-messages";
+        # If use multimachine, select correct needles to configure thunderbird.
+        if ($hostname eq 'client') {
+            send_key 'end';    # go to the bottom to see whole manual configuration
+            if (check_screen 'thunderbird_in-hostname-start-with-dot') {
+                record_info 'bsc#1191866';
```

Comment:
why not a softfail here?

Reply:
Did you see the bug ?!

Reply:
i did now. So it s invalid. there is not clear definition of usage form MOV but i guess it is fine in this case.

Reply:
It's common sense. There is no reason to spam bug because of soft fail.
I could just not add it at all, but it;s better than any `# comment` to understand why it is done.